ocean_data_parser/read/vocabularies/seabird_variable_attributes.json

```diff
@@ -2701,14 +2716,16 @@
     },
     "t090C": {
         "long_name": "Temperature",
-        "units": "deg C [ITS-90]",
+        "bodc_alternative_label": "CTDTmp90",
+        "units": "deg C",
```

Comment:
If we want to drop the scale from the units we may want to be consistent across all the terms.

Reply:
The comment I shared about the unit above would be applied here as well.

Reply:
Yeah ok I agree I think I'll move that to a scale attribute, most of those terms should already be represented within the P01 term. Though some of the variables haven't been matched yet to a BODC P01